tests/common/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2019 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+cmake_minimum_required(VERSION 3.5)
+
+add_library(olp-cpp-sdk-tests-common-lib INTERFACE)
+
+target_link_libraries(olp-cpp-sdk-tests-common-lib
+        INTERFACE
+                gmock
+                olp-cpp-sdk-core
+)
+
+target_include_directories(olp-cpp-sdk-tests-common-lib
+        INTERFACE
+                .
```

Comment:
We don't use variables for list of targets include directories everywhere else. I believe hiding list of directories as well as list of source files behind some mutable variable is a bad practice because you newer realy know what is in there by just looking at it.

Reply:
So using ${CMAKE_CURRENT_SOURCE_DIR} instead of `.` is a bad practice?

Reply:
Usually in places where we use variables for e.g. sources we don't use `${CMAKE_CURRENT_SOURCE_DIR}`, instead there is some kind of `${OLP_SDK_SOMETHING_SOMETHING_SOURCES}`. I thought about them first.

I don't see any difference between `${CMAKE_CURRENT_SOURCE_DIR}` and `.` and don't have any preference for one over another.